.yarnrc.yml

```diff
@@ -1 +1,4 @@
 yarnPath: .yarn/releases/yarn-3.2.3.cjs
+enableGlobalCache: true
+nodeLinker: node-modules
```

Comment:
My `prettier-vscode` doesn't seem to be working properly.

Reply:
Did you try https://yarnpkg.com/getting-started/editor-sdks?